### DISCUSSION TITLE
Error 403 for "B and H" and Adorama

### DISCUSSION THREAD

**Discussion by gam1ngdad** [jef/streetmerchant#1805] [General]
Looking for recommendations to try to get rid of the 403 error for BandH and Adorama.  I don't have any issues with any other sites.  Thanks in advance StreetMerchant is awesome!

**Comment by nullp01n73r** [comment]
having same issue

**Comment by wagequit** [comment]
same issue.  also, gamestop

**Comment by northmatt** [comment]
for b&h: turn off headless mode, it's a captcha issue. Can either try to make a new Captcha Deterrent for b&h (one of the more simplier captchas as it's just holding down a button) or make fairly slow reloading time.

edit: also worth noting that I wouldn't be surprised if Adorama or Gamestop has the same issue.
